### ISSUE TITLE
[FEATURE] Allow user to authenticate only once

### ISSUE THREAD

**Issue by askmon** [tq1/node-push-sender#16]
## User Story

As an user
I want to be able to authenticate only once
So that I don't pass authentication every time I send a push
## Tasks Checklist
- [ ] Change node-push-sender to get authentication object on require.
- [ ] Maintain compatibility
- [ ] Update docs (docs?) 
## Definition of Done

User should be able to pass auth only once and it still work

**Comment by askmon** [comment]
Duplicate of https://github.com/tq1/node-push-sender/issues/19
